LoopKit.xcodeproj/project.pbxproj

```diff
@@ -749,7 +748,14 @@
 		E96DCB5824AEF50F007117BC /* SuspendThresholdInformationView.swift in Sources */ = {isa = PBXBuildFile; fileRef = E96DCB5724AEF50F007117BC /* SuspendThresholdInformationView.swift */; };
 		E96DCB5A24AF74AC007117BC /* SuspendThresholdEditor.swift in Sources */ = {isa = PBXBuildFile; fileRef = E96DCB5924AF74AC007117BC /* SuspendThresholdEditor.swift */; };
 		E96DCB5E24AF7DC7007117BC /* BasalRatesInformationView.swift in Sources */ = {isa = PBXBuildFile; fileRef = E96DCB5D24AF7DC7007117BC /* BasalRatesInformationView.swift */; };
+		E9CDA72724E3A4BA0049B0D5 /* OutsideDoseEvent.swift in Sources */ = {isa = PBXBuildFile; fileRef = E9CDA72624E3A4BA0049B0D5 /* OutsideDoseEvent.swift */; };
+		E9CDA72824E3A4BA0049B0D5 /* OutsideDoseEvent.swift in Sources */ = {isa = PBXBuildFile; fileRef = E9CDA72624E3A4BA0049B0D5 /* OutsideDoseEvent.swift */; };
+		E9CDA72A24E3A4F60049B0D5 /* PersistedOutsideDoseEvent.swift in Sources */ = {isa = PBXBuildFile; fileRef = E9CDA72924E3A4F60049B0D5 /* PersistedOutsideDoseEvent.swift */; };
+		E9CDA72B24E3A4F60049B0D5 /* PersistedOutsideDoseEvent.swift in Sources */ = {isa = PBXBuildFile; fileRef = E9CDA72924E3A4F60049B0D5 /* PersistedOutsideDoseEvent.swift */; };
```

Comment:
I have no clue why these are showing up as doubles, but the build does fail when I try to manually remove them